src/modules/videoconference/VideoConferenceModule/Toolbar.cpp

```diff
@@ -256,7 +257,7 @@ void Toolbar::show(std::wstring position, std::wstring monitorString)
         else //"Top right corner" or non-present
         {
             positionX = monitorInfo.right() - overlayWidth - BORDER_OFFSET;
-            positionY = monitorInfo.top() + BORDER_OFFSET;
+            positionY = monitorInfo.top() + TOP_RIGHT_BORDER_OFFSET;
```

Comment:
Nit picking, but why is it called TOP_RIGHT when it's only for Y offset? :)

Reply:
because it's only used to specifically for the Top right corner, not for other corners :)